service/collector/sp_expiration.go

```diff
@@ -13,16 +13,10 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
```

Comment:
@whites11 I changed the SP expiration collector to make it reuse some of the already existing structs, so that it looks similar to other collectors. Can you take a look just in case I missed something?